en/xml/fo/topic.graphic-pageheader.xml

```diff
@@ -0,0 +1,266 @@
+        &lt;xsl:param name="position" select="''"/>
+        &lt;xsl:param name="gentext-key" select="''"/>
+        &lt;fo:block><co xml:id="co.topic.graphic-pageheader.block"/>
+            &lt;!-- sequence can be odd, even, first, blank -->
+            &lt;!-- position can be left, center, right -->
+            &lt;xsl:choose>
+                &lt;xsl:when test="$sequence = 'blank'">
+                    &lt;!-- nothing --><co xml:id="co.topic.graphic-pageheader.blank"/>
+                &lt;/xsl:when>
+
+                &lt;xsl:when test="$position='left'">
+                    &lt;!-- Same for odd, even, empty, and blank sequences -->
+                    &lt;!-- nothing -->
+                &lt;/xsl:when>
+
+                &lt;xsl:when test="($sequence='odd' or $sequence='even') and $position='center'">
+                    &lt;!-- nothing -->
+                &lt;/xsl:when>
+
+                &lt;xsl:when test="$position='center'"><co xml:id="co.topic.graphic-pageheader.center"/>
+                    &lt;!-- nothing for empty and blank sequences -->
+                &lt;/xsl:when>
+
+                &lt;xsl:when test="$position='right'"><co xml:id="co.topic.graphic-pageheader.right"/>
+                    &lt;!-- Same for odd, even, empty, and blank sequences -->
+                    &lt;fo:external-graphic content-height="1in"><co xml:id="co.topic.graphic-pageheader.height"/>
+                       &lt;xsl:attribute name="src"><co xml:id="co.topic.graphic-pageheader.attribute"/>
+                            &lt;xsl:call-template name="fo-external-image"><co xml:id="co.topic.graphic-pageheader.template"/>
+                                &lt;xsl:with-param name="filename" select="concat($img.src.path, $header.image.filename)"/><co xml:id="co.topic.graphic-pageheader.file"/>
+                            &lt;/xsl:call-template>
+                       &lt;/xsl:attribute>
+                    &lt;/fo:external-graphic>
+                &lt;/xsl:when>
+
+                &lt;xsl:when test="$sequence = 'first'">
+                    &lt;!-- nothing for first pages -->
+                &lt;/xsl:when>
+
+                &lt;xsl:when test="$sequence = 'blank'">
+                    &lt;!-- nothing for blank pages -->
+                &lt;/xsl:when>
+            &lt;/xsl:choose>
+        &lt;/fo:block>
+    &lt;/xsl:template>
+&lt;/xsl:stylesheet></programlisting>
+                    <calloutlist>
+                        <callout arearefs="co.topic.graphic-pageheader.path">
+                            <para>This setting is global for your pictures,
+                                logos, graphics. Keep in mind that the
+                                    <tag>imagedata</tag> command does not need a
+                                path if all files are in the same folder. You
+                                can write something like this:</para>
+                            <para>&lt;imagedata fileref="pic02.pdf"/>
```

Comment:
Use `screen` instead of `para`.

Reply:
No longer appreciale, because we are now using logo.src.path instead of img.src.path.